Uint16 marker_num
---
geometry_msgs/PoseStamped position
